packages/veritone-csp-generator/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+const env = require('veritone-dev-env');
+
+module.exports = Object.assign({}, env.eslintReact, {
+  globals: Object.assign({}, env.eslintReact.globals, { module: true }),
```

Comment:
since you're using rollup now, you can write your exports with es6 syntax, so the `module` global wont be needed -- i think you can delete this line